asn/asn.go

```diff
@@ -0,0 +1,60 @@
+package asn
+
+import (
+	"net"
+	"strconv"
+	"strings"
+
+	asnmap "github.com/projectdiscovery/asnmap/libs"
+	"github.com/projectdiscovery/mapcidr"
+)
+
+type ASNClient struct {
+	client *asnmap.Client
+}
+
+func New() ASNClient {
+	return ASNClient{
+		client: asnmap.NewClient(),
+	}
+}
+
+// GetCIDRsForASNNum returns the slice of cidrs for given ASN number
+// accept the ASN number like 'AS15133' and returns the CIDRs for that ASN
+func (c *ASNClient) GetCIDRsForASNNum(value string) []*net.IPNet {
```

Comment:
I think here we should return an error if the `value` argument doesn't have the required length